seed/api/v2_1/views.py

```diff
@@ -17,7 +17,7 @@
 from rest_framework import status
 from rest_framework.decorators import detail_route
 
-from seed.building_sync.building_sync import BuildingSync
+from seed.building_sync.building_sync import BuildingSync  # TODO: Wut?
```

Comment:
I couldn't tell what happened during conflict resolution here and in the urls file.  I'll have to check the commits on develop and see.